templates/index.html

```diff
@@ -582,11 +661,11 @@ <h3 class="p-text--small-caps">What's new</h3>
 </div>
 
 {% with %}
-  {% set formid = "5311" %}
-  {% set returnURL = "https://canonical.com#success" %}
-  {% include "partners/partial/_homepage-form.html" %}
+{% set formid = "5311" %}
+{% set returnURL = "https://canonical.com#success" %}
+{% include "partners/partial/_homepage-form.html" %}
```

Comment:
This was also correctly indented, because `{% set formid = "5311" %}` is within the scope of `{% with %}`